src/main/java/seedu/address/logic/commands/DeleteInterviewCommand.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.interview.Interview;
+
+/**
+ * Command to handle deleting interviews from the address book (using the interview index).
+ * Adapted from AB3's "DeleteCommand" class
+ */
+public class DeleteInterviewCommand extends Command {
+
+    public static final String COMMAND_WORD = "delete-interview";
```

Comment:
Perhaps the command word should be delete-i based on the format of the other commands and the user guide.

Reply:
Good catch on the standardization, fixed.